app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable, :confirmable
+  include DeviseTokenAuth::Concerns::User
+
+  enum gender: %i[male female]
```

Comment:
Minor comment, it is recommended the use of the hash syntax for enums (https://github.com/rootstrap/tech-guides/blob/master/ruby/rails.md#macro-style-methods)